views/js/controller/list/index.js

```diff
@@ -87,125 +88,134 @@ define([
         findListContainer($(this).attr('id')).toggleClass('with-uri');
     }
 
-    return {
+    function handleEditList (uriList) {
+        const uri = uriList || $(this).data('uri');
```

Comment:
We need to fix the build https://codeclimate.com/github/oat-sa/extension-tao-backoffice/pull/114